Description: add encoding directive to pod, needed by pod2man
Origin: vendor
Author: gregor herrmann <gregoa@debian.org>
Last-Update: 2014-04-03

--- a/Changes.pod
+++ b/Changes.pod
@@ -1,3 +1,4 @@
+=encoding UTF-8
 
 =head1 WHATS NEW IN SCRIPTALICIOUS
 
